### ISSUE TITLE
remove the hyperlinks in the result pages

### ISSUE THREAD

**Issue by epaule** [epaule/nameserver2#36]
... so Mary Ann can copy and paste

**Comment by Paul-Davis** [comment]
No, I think it's useful to be able to check the gene has received the appropriate history in the table.......maybe have the hyperlink and print the ID as well

"WBGene00000001" - WBGene00000001
